src/sentry/api/serializers/models/group.py

```diff
@@ -787,19 +787,25 @@ def _execute_seen_stats_query(
         }
         user_counts = {item_id: value["count"] for item_id, value in seen_data.items()}
         last_seen = {item_id: value["last_seen"] for item_id, value in seen_data.items()}
-        times_seen = {item_id: value["times_seen"] for item_id, value in seen_data.items()}
-        if not environment_ids:
-            first_seen = {item_id: value["first_seen"] for item_id, value in seen_data.items()}
+        if not environment_ids and not start and not end and not conditions:
```

Comment:
I think maybe we need a comment here explaining that mostly we care about the `first_seen` just for the global stats, and that it's unused in the other ones. Otherwise people might think that they need to keep fetching `first_seen` from snuba in the future.